### DISCUSSION TITLE
Remove "add a category" button when category field has reached branch limit

### DISCUSSION THREAD

**Discussion by edhebert** [craftcms/cms#9682] [Ideas] [answered]
When interacting with a category field in the CP, please consider removing the "add a category" button once the category field has reached its branch limit.  Current workflow still allows the user to click the "add" button and seemingly select another category. Thankfully that category isn't added to the list if the limit is met - however, there also is no prompt that a selection limit has been reached.  This creates confusion as content editors have no indication why their content isn't being accepted.

**Comment by brandonkelly** [comment, marked as answer]
This is already covered by #2423.
